src/TemplateEditor.js

```diff
@@ -209,12 +264,25 @@ class TemplateEditor extends React.Component {
       printable,
       required,
       selectedCategory,
+      name,
+      editAsHtml,
+      intl: { formatMessage }
     } = this.props;
 
     const invalid = (touched || submitFailed) && !valid && !showTokensDialog;
 
     const appliedValue = sanitize(value);
 
+    const extraButton = !editAsHtml ? undefined : (
+      <Button
+        bottomMargin0
+        onClick={this.openTokenDialog}
+        aria-label={formatMessage({ id: 'stripes-template-editor.toolbar.token' })}
+      >
+        {'{ }'}
+      </Button>
+    );
```

Comment:
Sure, sure, it makes sense for `ControlHeader` to receive a generic `extraButton` prop, but here in `TemplateEditor` that "extra" button is very specifically the `showTokensButton` or `showTokensTrigger` or some such. Would sth like this be better?
```
const showTokensButton = <Button onClick={this.openTokenDialog}>{'{ }'}</Button>
return  <ControlHeader extraButton={showTokensButton} />
```